Cakes-Co/index.html

```diff
@@ -7,10 +7,145 @@
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header>
+        <img src="./images/cakehouse-icon.jpeg" alt="cake-icon" height="40" width="40">
+        <h2>My Halal bakery<span>est. since 2000</span></h2>
+        <p>Elevating sweetness into a whole new level</p>
+
+        <nav>
+            <ul>
+                <li class="nav-item"><a href="#">Home</a></li>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="#">Our Products</a></li>
+                <li class="nav-item"><a href="#">News</a></li>
+                <li class="nav-item"><a href="#">Contact Us</a></li>
+                <li class="nav-item"><a href="#">Help</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <h2>Pleasure and taste in one place</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse ultricies rutrum orci. Nulla
+                placerat, metus vel
+                fermentum suscipit, lacus nisi tempus lacus, sit amet commodo quam urna in libero. Maecenas at eros
+                malesuada, lacinia
+                turpis a, congue felis. Duis eu purus nisi. Cras mi ligula, mollis eget ullamcorper eu, elementum ut
+                risus. Nullam
+                pretium vitae tellus id molestie.</p>
+            <p>Making sure that the muslim community feed their bellies with halal pastries</p>
+        </section>
+        <section>
+            <img src="./images/chocolate cake.jpg" alt="chocolate-cake" height="500px" width="500px">
+            <h3>Chocolate cake</h3>
+            <p>Duis sit amet rhoncus dui, sit amet feugiat magna. Curabitur eu ligula in orci eleifend dictum nec quis
+                augue. Vivamus
+                sit amet felis sodales augue condimentum cursus. Quisque sem erat, congue vel dui a, aliquam porta est.
+                Phasellus quis
+                orci turpis. Morbi pellentesque nisl vitae justo mattis sodales. Suspendisse sagittis et leo eget
+                rhoncus.</p>
+        </section>
+        <section>
+            <section>
+                <img src="./images/fruit-cake.avif" alt="chocolate-cake" height="500px" width="500px">
+                <h3>fruit cake</h3>
+                <p>Duis sit amet rhoncus dui, sit amet feugiat magna. Curabitur eu ligula in orci eleifend dictum nec
+                    quis augue.
+                    Vivamus
+                    sit amet felis sodales augue condimentum cursus. Quisque sem erat, congue vel dui a, aliquam porta
+                    est. Phasellus
+                    quis
+                    orci turpis. Morbi pellentesque nisl vitae justo mattis sodales. Suspendisse sagittis et leo eget
+                    rhoncus.</p>
+            </section>
+            <section>
+                <img src="./images/pistachio layer cake.jpg" alt="chocolate-cake" height="500px" width="500px">
```

Comment:
This is the third image with the same "chocolate-cake" alt element - this will be a very boring website for a person with a visual impairment to navigate.